optuna/study/study.py

```diff
@@ -951,7 +951,7 @@ def add_trials(self, trials: Iterable[FrozenTrial]) -> None:
             .. testcode::
 
                 import optuna
-                from optuna.distributions import UniformDistribution
+                from optuna.distributions import FloatDistribution
```

Comment:
```suggestion
```

Hmm, I just found it is not used in anywhere.

Reply:
Good catch! 922752436d2df76bf7e5d5b105ea9640e6e673da.